instructor-notes/stacking.md

```diff
@@ -5,13 +5,15 @@
 Before the live demo, slides will have covered only the concept of `git` branch stacking, as well as why it's a challenging thing to do in forks.
 There are no new `git` commands that need to be introduced for this live demo.
 
+Note that the second section of the demo should be conducted from a _fork_ of the demo repository.
+Depending on the given workshop audience, this fork can be set up in advance of the workshop, or you can show creating and cloning the fork during the workshop, which includes setting the upstream repository: `git remote add upstream git@github.com:<upstream account>/<upstream repo>`.
```

Comment:
If you're going to set it up live, using `git remote -v` before and after per the [GitHub docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-repository-for-a-fork) seems like a good move!

Reply:
ohh :100: this is my move always!